src/legacy/core_plugins/kibana/public/discover/index.html

```diff
@@ -122,6 +90,15 @@ <h1 id="kui_local_breadcrumb" class="kuiLocalBreadcrumb" ng-if="opts.savedSearch
               </span>
             </button>
 
+            <div class="dscResultCount">
+              <span data-test-subj="discoverQueryHits" class="kuiLocalBreadcrumb__emphasis">{{(hits || 0) | number:0}}</span>
```

Comment:
Instead of using this legacy class, just make the element a `strong` tag
```suggestion
              <strong data-test-subj="discoverQueryHits">{{(hits || 0) | number:0}}</strong>
```